osdep/timer-linux.c

```diff
@@ -35,15 +40,27 @@ void mp_sleep_ns(int64_t ns)
 uint64_t mp_raw_time_ns(void)
 {
     struct timespec tp = {0};
-    int ret = -1;
-#if defined(CLOCK_MONOTONIC_RAW)
-    ret = clock_gettime(CLOCK_MONOTONIC_RAW, &tp);
-#endif
-    if (ret)
-        clock_gettime(CLOCK_MONOTONIC, &tp);
+    clock_gettime(clk_id, &tp);
     return MP_TIME_S_TO_NS(tp.tv_sec) + tp.tv_nsec;
 }
 
 void mp_raw_time_init(void)
 {
+    static const clockid_t clock_ids[] = {
+#if defined(CLOCK_MONOTONIC_RAW)
+        CLOCK_MONOTONIC_RAW,
+#endif
+        CLOCK_MONOTONIC,
+        CLOCK_REALTIME,
```

Comment:
I don't think we really want CLOCK_REALTIME here even as a fallback.

Reply:
Removed.